### PR TITLE
Optimize for mingz, remove aliases. 

### LINKED ISSUE #30: The test suite fails on OS X 10.10
The `fs-events` package refuses to install, and the test runner (`npm run check`) crashes  with

```
[SERVER] [error] Error: Missing /Users/pygy/src/little-loader/node_modules/selenium-standalone/.selenium/chromedriver/2.18-x64-chromedriver
  1) "before all" hook
```

Here's [the full log](https://gist.github.com/pygy/9efa5470bcfb38ba28a1), including early successes then a crash.

This is on the master branch, freshly cloned.

```
~/s/little-loader git:master ❯❯❯ node -v
v5.1.1
~/s/little-loader git:master ❯❯❯ npm -v
3.5.2
```

### PR DESCRIPTION
**Pull request by ryan-roemer**
Fixes #30

This is really @pygy 's PR, but I'm reopening so that Sauce Labs credentials work and such.

We'll wait for @pygy 's approval of "I'm done with optimizations", then do final review.

/cc @exogen

### CODE REVIEW

**Comment by pygy** [comment]
At least SauceLabs, Travis and Coveralls are happy now. Good :-)

**Comment by ryan-roemer** [comment]
@pygy -- Any other optimizations you'd like, or are we good to go with current iteration here?

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ A lightweight, IE8+ JavaScript loader that is **actually tested**...
 * **Tested** all the way down to IE8
 * Reliably **calls back** after script loads
 * Captures script load **errors** down to IE8
-* Really, really **small** (clocking in at `~865` minified bytes)
+* Really, really **small** (clocking in at `~474` minified + gzipped bytes)
```
Comment (baer) [inline README.md]: :clap:

**Comment by baer** [comment]
This LGTM! Nice work.

**Comment by pygy** [comment]
There's the pending question of how can `head` not exist unless it has been deleted. I'm not aware of a scenario where this occurs (see #32 for a minimal example), but @exogen says it can happen.

Otherwise, yes, #32 is ready to merge (please ;-).

**Comment by ryan-roemer** [comment]
@pygy -- The `head` change seems a bit risky, so let's defer that for now. Can you perhaps cover in a ticket that @exogen can comment on?

I'm going to merge this PR (with attribution to you when I do the HISTORY) and close yours. Thanks again!

**Comment by ryan-roemer** [comment]
Released as `little-loader@0.1.1`

**Comment by pygy** [comment]
I would have preferred appearing in the git log as well... Too late I guess.

**Comment by ryan-roemer** [comment]
Ah snap, forgot about that. I'll give you the final diff of a PR to rebase on next time to merge in (I had a few extra doc updates I wanted in my PR). Sorry!

**Pull request merged**